filesystem/Upload.php

```diff
@@ -131,7 +131,26 @@ public function load($tmpFile, $folderPath = false) {
 			$this->errors[] = _t('File.NOFILESIZE', 'File size is zero bytes.');
 			return false;
 		}
-
+		
+		if (!empty($tmpFile['type'])) {
+            $type = $tmpFile['type'];
+            if (strpos($type, 'image/') !== false) {
+                $tmp_name = $tmpFile['tmp_name'];
+                try {
+                    $test = new ImagickBackend($tmp_name);
```

Comment:
What if I use GD and don’t have `Imagick` installed? Won’t this remove all of my images?